doc/installation.rst

```diff
@@ -76,176 +91,183 @@ On macOS, open the disk image and drag Spyder to your :guilabel:`Applications` f
 
-You can then run Spyder by the same methods as above, except make sure to select the start menu shortcut with ``(spyder-env)`` in the name, select the ``spyder-env`` environment on the left before launching it with Navigator, or type ``conda activate spyder-env`` before launching it on the command line.
+For a minimal install without the optional functionality and integration with the above packages, you can instead run:
 
-.. image:: /images/installation/installation-conda-install.gif
-   :alt: Running Spyder installation with conda
+.. code-block:: shell
 
-For more information on this approach, and using Spyder with your existing Python environments and packages, please see our `Guide to working with packages and environments in Spyder`_.
+   conda create -c conda-forge -n spyder-env spyder
 
-.. _Guide to working with packages and environments in Spyder: https://github.com/spyder-ide/spyder/wiki/Working-with-packages-and-environments-in-Spyder
+This installs Spyder into a new environment called ``spyder-env``, using the more up-to-date, community-run Conda-Forge channel.
+To make sure future installs/updates in this environment also use Conda-Forge and are faster and more reliable, make sure to set it as your environment's default channel with strict channel priority enabled, if this isn't the case already (as it is with Miniforge/Mambaforge or if you've manually configured it):
 
+.. code-block:: shell
 
+   conda activate spyder-env
+   conda config --env --add channels conda-forge
+   conda config --env --set channel_priority strict
 
-===================
-Alternative methods
-===================
+Here's a summary of the main steps.
 
-.. caution::
+.. image:: /images/installation/installation-conda-install.gif
+   :alt: Running Spyder installation with conda
 
-   While we offer alternative Spyder installation options for users who desire them, we currently lack the resources to offer individual assistance for problems specific to installing via these alternative distributions.
-   Therefore, we recommend you switch to our :ref:`standalone installers<standalone_installers_ref>` (Windows and macOS) or :ref:`anaconda_install_ref` if you encounter installation issues you are unable to solve on your own.
 
+.. _install-conda-running:
 
-Windows
-~~~~~~~
+Running with Conda
+------------------
 
-Spyder is included in the `WinPython`_ scientific Python distribution (although Anaconda's ``conda`` package and environment manager is not).
-You can use Spyder immediately after installing, just like with Anaconda.
+You can then run Spyder by the same methods :ref:`as with Anaconda <install-anaconda-running>`, except make sure to launch the start menu shortcut with ``(spyder-env)`` in the name, select the ``spyder-env`` environment on the left before launching it with Navigator, or type ``conda activate spyder-env`` before launching it on the command line.
```

Comment:
```suggestion
You can then run Spyder by the same methods :ref:`as with Anaconda <install-anaconda-running>`, except that you need to make sure to launch the Start menu shortcut with ``(spyder-env)`` in the name, select the ``spyder-env`` environment on the left before launching it with Navigator, or type ``conda activate spyder-env`` before launching it on the command line.
```

* I think this would make the sentence easier to read for non-native English speakers, although I'm not sure if it is correct.
* Start menu is capitalized in other places (and I think that's correct).

Reply:
Sure; its a little belabored, but it is perfectly correct and clearer to me as well :+1: